### PR TITLE
updated name to excel 365 and made specific callout in authorization …

### PR DESCRIPTION
**Pull request by wrp801**
Updated Excel Docs and authorization guide

### CODE REVIEW

File: docs/blueprint-library/microsoft-excel/microsoft-excel-download-data-from-workbook-to-shipyard.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Fleet [YAML Editor](../../reference/fleets/yaml-editor.md).
 
 ```yaml
 source:
-  blueprint: Microsoft Excel - Download Data from Workbook to Shipyard
+  blueprint: Microsoft Excel (365) - Download Data from Workbook to Shipyard
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/microsoft-excel/microsoft-excel-download-data-from-workbook-to-shipyard.md:45]: I don't think the () are needed

Comment (wrp801) [reply docs/blueprint-library/microsoft-excel/microsoft-excel-download-data-from-workbook-to-shipyard.md:45]: This was Steven's suggestion

**Comment by wrp801** [comment]
Renamed to Microsoft 365 Excel

**Review by johnathan-rodriguez — DISMISSED** [review]

File: docs/blueprint-library/microsoft-excel/microsoft-excel-upload-data-to-workbook.md [hunk 2]
```diff
@@ -46,7 +47,7 @@ Fleet [YAML Editor](../../reference/fleets/yaml-editor.md).
 
 ```yaml
 source:
-  blueprint: Microsoft Excel - Upload Data to Workbook from Shipyard
+  blueprint: Microsoft Excel (365) - Upload Data to Workbook from Shipyard
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/microsoft-excel/microsoft-excel-upload-data-to-workbook.md:50]: Missed this

**Pull request merged**